examples/singularity/README.md

```diff
@@ -0,0 +1,38 @@
+# Run jobs inside a  singularity container
+
+The example has a single rule, which is run in a singularity container.  To run this example you will need singularity installed and available in your `$PATH` (i.e the shell command `which singularity` must return the path to the singularity executable).
+
+## Running the example
+```sh
+# set username and password for whatever service you  GitHub PAT
+export SINGULARITY_DOCKER_USERNAME=
+export SINGULARITY_DOCKER_PASSWORD=Put_Your_GitHub_PAT_Here
```

Comment:
I found this early mention of a GitHub PAT confusing. How about re-ordering the instructions to follow the order of increasing complexity?

* Here's how to run the command (ie just the call to snakemake)
* But if you get this Docker error about a rate limit, then you'll need to authenticate first with your Docker credentials
* And another option for authentication would be to use GitHub